src/imu/sensors/temperature.hpp

```diff
@@ -0,0 +1,23 @@
+#ifndef SRC_TEMPERATURE_HPP_
+#define SRC_TEMPERATURE_HPP_
+
+#include "temperature_virtual_interface.hpp"
+
+namespace imu {
+
+class Temperature final : public TemperatureInterface {
```

Comment:
is called TemperatureSensor in class diagram

Reply:
Adjusted in diagram. Should be suitable to other sensors.